docs/concepts/trading-framework/fees.md

```diff
@@ -59,19 +59,29 @@ The fees paid are transferred to a liquidity fee account, and distributed to eac
 [How the liquidity fee is set](../liquidity/rewards-penalties.md#determining-the-liquidity-fee-percentage)
 :::
 
+### Network treasury fee
+The network treasury fee is collected from takers and transferred to the [network treasury](../assets/accounts.md#network-treasury-accounts).
+
+The amount charged for this fee is set by a network parameter.
+
+### Buyback fee
+The buyback fee is collected from takers and is used to buy governance tokens on a spot market via regular auctions. If there is no relevant spot market, and the fee is set to a value higher than 0, the accrued fee payments are transferred to the [network treasury](../assets/accounts.md#network-treasury-accounts).
```

Comment:
```suggestion
The buyback fee is collected from takers and is used to buy specific tokens on a spot market through Protocol Automated Purchase programs.
```

The bit about relevant spot markets is not true, if there are no programs active the funds will just sit in the buyback account. They can always be transferred later through govenrance.